Makefile

```diff
@@ -14,11 +14,14 @@
 
 IMAGE=amazon/aws-ebs-csi-driver
 VERSION=0.1.0-alpha
+GIT_COMMIT?=$(shell git rev-parse HEAD)
+BUILD_DATE?=$(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
+LDFLAGS?="-X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.driverVersion=${VERSION} -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.gitCommit=${GIT_COMMIT} -X github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver.buildDate=${BUILD_DATE}"
```

Comment:
Is it possible to create a variable for `github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/driver` and refer it from within `LDFLAGS`? This line is long and, when people makes changes, it could be error-prune if we have it in three places